### ISSUE TITLE
Ability to select multiple attendants in one appointment.

### ISSUE THREAD

**Issue by alextselegidis** [alextselegidis/easyappointments#261]
It must be possible for a service to accept have an attendants-validation type where a maximum number of attendants are set per slot. Customers will be able to set the number of attendants per appointment and the final booking will validate if the maximum capacity was reached.

**Comment by obalsoy** [comment]
Hello. In version 1.2, you announce that multiple attendants are now allowed for the same spot.
How do I activate or configure that feature?
Thanks.

**Comment by alextselegidis** [comment]
It's the multiple-attendants property of the services.

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

**Comment by xcsn2010** [comment]
This should be specified as attendant-capacity in back-end at service level, but it should be front-end input from the client that validates how many will come, right?

**Comment by alextselegidis** [comment]
@xcsn2010 Yes you are right. Basically the user that makes the booking will be able to enter data for additional attendants. 

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>
